src/lambdas/initiateAthenaQuery/createQuerySql.ts

```diff
@@ -105,3 +109,17 @@ const formatWhereStatment = (
 
   return `${identifierType} IN (${whereStatementsArray.join(', ')})`
 }
+
+const formatDateFrom = (dateFrom: string): string => {
+  const splitDateFrom = dateFrom.split('-')
+  splitDateFrom.push('00')
+  const formattedDateFrom = splitDateFrom.join('/')
```

Comment:
You could just return the new join date instead of storing it in a const and returning it
` return splitDateFrom.join('/')`

Reply:
Thanks, will do